src/ports/analyticsDayData/utils.ts

```diff
@@ -77,3 +90,58 @@ export function getTimestampFromTimeframe(timeframe: AnalyticsTimeframe) {
       return 0
   }
 }
+
+// Rentals
+
+export function getRentalsAnalyticsDayDataQuery(
+  filters: AnalyticsDayDataFilters
+) {
+  const { from } = filters
+
+  const where: string[] = []
+
+  if (from) {
+    where.push(`date_gt: ${Math.round(from / 1000)}`)
+  }
+
+  return `
+    query AnalyticsDayData {
+      analytics: analyticsDayDatas(where: { ${where.join('\n')} }) {
+        id
+        date
+        rentals
+        volume
+        lessorEarnings
+        feeCollectorEarnings
+      }
+    }
+  `
+}
+
+export function getRentalsAnalyticsTotalDataQuery() {
+  return `
+    query AnalyticsTotalData {
+      analytics: analyticsTotalDatas {
+        id
+        rentals
+        volume
+        lessorEarnings
+        feeCollectorEarnings
+      }
+    }
+  `
+}
+
+export function mapRentalsAnalyticsFragment(fragment: any): AnalyticsDayData {
```

Comment:
We do know the rental fragment type here, right? Can we type it?

Reply:
Can we add tests to this new function?

Reply:
Typed it, writing the tests...

Reply:
Wrote test